sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -97,7 +98,9 @@ def get_estimated_size_and_observables(self, value, nested=False):
 
 
 class SimpleCoderImpl(CoderImpl):
-  """Subclass of CoderImpl implementing stream methods using encode/decode."""
+  """For internal use only; no backwards-compatibility guarantees.
```

Comment:
Just wanted to check if the DocString still renders correctly as I thought the first line acted as a summary.